libs/domains/clusters/feature/src/lib/cluster-action-toolbar/cluster-action-toolbar.tsx

```diff
@@ -174,6 +199,15 @@ function MenuOtherActions({
         >
           Get Kubeconfig
         </DropdownMenu.Item>
+        {cluster.kubernetes === 'SELF_MANAGED' && (
+          <DropdownMenu.Item
+            key="3"
```

Comment:
key is useless here

Reply:
https://github.com/Qovery/console/pull/1117/commits/d7045fef0f3a252a41db3458a728f159c0c1ef75